net-analyzer/arpwatch/files/0004-Ignore-unused-results.patch

```diff
@@ -0,0 +1,17 @@
+Prevent warnings about unused return codes of setgid/setuid.
```

Comment:
I can't see the value in this patch. The point in the warning is to signal that someone should handle it properly. Do we have to handle that downstream (by fixing it properly)? No, but silencing it has no benefit, and only downsides (like confusing someone who might be debugging and look for a warning that isn't there).

Just leave it out and do nothing about it, maybe an upstream report, but nothing more.